### PR TITLE
Split out sample prior from createRunPath

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [ ] Added appropriate release note label
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by oyvindeide** [comment]
test required please

File: src/ert/_c_wrappers/enkf/enkf_main.py [hunk 1]
```diff
@@ -316,26 +316,32 @@ def getHistoryLength(self) -> int:
     def getWorkflowList(self) -> ErtWorkflowList:
         return self.resConfig().ert_workflow_list
 
-    def initRun(self, run_context: "RunContext", parameters: List[str] = None):
+    def sample_prior(
+        self,
+        storage: "EnkfFs",
+        active_realizations: List[int],
+        parameters: List[str] = None,
+    ):
```
Comment (frode-aarstad) [inline src/ert/_c_wrappers/enkf/enkf_main.py:324]: -> None: ?

File: src/ert/gui/tools/manage_cases/case_init_configuration.py [hunk 2]
```diff
@@ -108,14 +108,13 @@ def addInitializeFromScratchTab(self):
...
-                mask[int(member.strip())] = True
             case_manager = self.ert.storage_manager
             sim_fs = case_manager.current_case
-            run_context = RunContext(sim_fs=sim_fs, mask=mask)
-            self.ert.initRun(run_context, parameters=parameters)
+            self.ert.sample_prior(
+                sim_fs,
+                [int(i) for i in members_model.getSelectedItems()],
+                parameters=parameters,
+            )
```
Comment (frode-aarstad) [inline src/ert/gui/tools/manage_cases/case_init_configuration.py:117]: perhaps add names for all parameters, storage, active_reals ..

**Review by frode-aarstad — APPROVED** [review]
Just a few minor comments. Other than that LGTM

**Pull request merged**